### PR TITLE
Addresses #222 Recommendations for using sudo.

### PR DESCRIPTION
**Pull request by daobrien**
Would very much appreciate review by a techie person, both because of the nature of the material and also because it's a public guide and this commit touches on how we configure our courses to some degree.

### CODE REVIEW

File: en-US/Design.xml [hunk 1]
```diff
@@ -371,6 +372,89 @@ $ vi myFile.txt
...
+      <section><title>Exceptions</title>
+        <para>
+          Some courses are specifically designed to teach <command>sudo</command> and its variations, the use of the related files, such as <filename>/etc/sudoers</filename> and so on.
+          For these courses, use the required variation for the topic being taught.
+        </para>
+        <section><title>Ansible Courses</title>
+          <itemizedlist>
+            <listitem>
+              <para>
+                Ansible courses typically use a <systemitem>devops</systemitem> user with passwordless <command>sudo ALL=ALL(ALL)</command> access on managed nodes to enable the use of <command>become</command> without a <command>become</command> password as <systemitem>root</systemitem> to do anything.
```
Comment (mweetman-redhat) [inline en-US/Design.xml:438]: I think `command` should be replaced with `code` tags, and include `NOPASSWD:` if it needs to be passwordless.

**Review by mweetman-redhat — CHANGES_REQUESTED** [review]

File: en-US/Design.xml (hunk 1 above)
Comment (daobrien) [reply en-US/Design.xml:438]: I can change the tags, no problem, but perhaps you could do a `Ctrl+G` and make a suggestion about the NOPASSWD: bit to make sure I get it right? thanks

Comment (mweetman-redhat) [reply en-US/Design.xml:438]:
```suggestion
                Ansible courses typically use a <systemitem>devops</systemitem> user with passwordless sudo access (<code>devops	ALL=(ALL)	NOPASSWD: ALL</code>) on managed nodes to enable the use of <command>become</command> without a <command>become</command> password as <systemitem>root</systemitem> to do anything.
```

**Comment by bonnevil** [comment]
I know you resolved this before I had a chance to review, but another useful classroom setup point is that we can customize the /etc/sudoers configuration for a particular course by adding files to /etc/sudoers.d, which avoids the need to make edits to the default /etc/sudoers file.

**Comment by daobrien** [comment]
> I know you resolved this before I had a chance to review, but another useful classroom setup point is that we can customize the /etc/sudoers configuration for a particular course by adding files to /etc/sudoers.d, which avoids the need to make edits to the default /etc/sudoers file.

Is something you'd def like to document, and if so do you want it in the public style guide? I feel perilously close to the border of Tech Writing Style Guide and System Setup Best Practices.

**Pull request merged**